hail/python/hail/linalg/blockmatrix.py

```diff
@@ -999,8 +1000,15 @@ def sparsify_triangle(self, lower=False, blocks_only=False):
 
         return self.sparsify_band(lower_band, upper_band, blocks_only)
 
-    @typecheck_method(starts=oneof(sequenceof(int), np.ndarray),
-                      stops=oneof(sequenceof(int), np.ndarray),
+    @typecheck_method(intervals=expr_tuple([expr_array(expr_int64), expr_array(expr_int64)]),
+                      blocks_only=bool)
+    def _sparsify_row_intervals_expr(self, intervals, blocks_only=False):
+        return BlockMatrix(
+            BlockMatrixSparsify(self._bmir, intervals._ir,
+                                RowIntervalSparsifier(blocks_only)))
+
+    @typecheck_method(starts=oneof(sequenceof(int), np.ndarray, expr_tuple([exor_int64, expr_int64])),
```

Comment:
Typo, this says `exor_int64`

Reply:
oops, yeah (actually that was a change that I forgot to back out, so I backed it out. From when I was considering having both of them go through the same function.)